src/components/icons/controller.icons.ts

```diff
@@ -70,8 +71,39 @@ const getString = async (req: Express.Request, res: Express.Response, next: Expr
   }
 }
 
+const getFile = async (req: Express.Request, res: Express.Response, next: Express.NextFunction) => {
+  try {
+    const theme = req.query.theme as iconsTheme
```

Comment:
if this is a POST method then why are we taking theme in query params and not payload itself? There is no problem but for this we will need to take care of payload on params both on FE. ideally for POST request, all data should be in the body

Reply:
I will merge it, and fix it with other APIs in a separate PR.